src/query_generate.rs

```diff
@@ -543,7 +573,7 @@ fn generate_select_by_fk_query_code(
     select_code.push_str(&format!(
         "    pub async fn all_by_{}_{}<'e, E: PgExecutor<'e>>(executor: E, {}_{}: {}) -> Result<Vec<{}>> {{\n",
         to_snake_case(foreign_row_table_name),
-        foreign_row_column_name,
+        to_snake_case(column_name),
```

Comment:
I think this might need to be foreign_row_column_name still

Reply:
yeah, you're right. I've changed this because I was figuring out what was causing the duplicate functions.

Reply:
with `foreign_row_column_name`:
```rust
     pub async fn all_by_profile_id<'e, E: PgExecutor<'e>>(executor: E, profile_id: i64) -> Result<Vec<Friendship>> {
        query_as::<_, Friendship>(r#"SELECT * FROM "friendship" WHERE from_user = $1"#)
            .bind(profile_id)
            .fetch_all(executor)
            .await
    }
    pub async fn all_by_profile_id<'e, E: PgExecutor<'e>>(executor: E, profile_id: i64) -> Result<Vec<Friendship>> {
        query_as::<_, Friendship>(r#"SELECT * FROM "friendship" WHERE to_user = $1"#)
            .bind(profile_id)
            .fetch_all(executor)
            .await
    }
```

with `to_snake_case(column_name)`:
```rust
    pub async fn all_by_profile_from_user<'e, E: PgExecutor<'e>>(executor: E, profile_id: i64) -> Result<Vec<Friendship>> {
        query_as::<_, Friendship>(r#"SELECT * FROM "friendship" WHERE from_user = $1"#)
            .bind(profile_id)
            .fetch_all(executor)
            .await
    }
    pub async fn all_by_profile_to_user<'e, E: PgExecutor<'e>>(executor: E, profile_id: i64) -> Result<Vec<Friendship>> {
        query_as::<_, Friendship>(r#"SELECT * FROM "friendship" WHERE to_user = $1"#)
            .bind(profile_id)
            .fetch_all(executor)
            .await
    }
```

yeah I remember why I changed it :)